os/drivers/task_manager/task_manager_drv.c

```diff
@@ -43,6 +45,13 @@ static int taskmgr_ioctl(FAR struct file *filep, int cmd, unsigned long arg);
 static ssize_t taskmgr_read(FAR struct file *filep, FAR char *buffer, size_t len);
 static ssize_t taskmgr_write(FAR struct file *filep, FAR const char *buffer, size_t len);
 
+/*****************************************************************************
+ * Pre-processor Definitions
+ *****************************************************************************/
+#if defined(HAVE_TASK_GROUP) && !defined(CONFIG_DISABLE_PTHREAD) && defined(HAVE_GROUP_MEMBERS)
```

Comment:
2nd conditional is unnecesary. Because `HAVE_GROUP_MEMBERS` is enabled, only when `CONFIG_DISABLE_PTHREAD` is disabled.
```
/* In any event, we don't need group members if support for pthreads is disabled */

#ifdef CONFIG_DISABLE_PTHREAD
#undef HAVE_GROUP_MEMBERS
#endif
```